sdk/formrecognizer/azure-ai-formrecognizer-perf/src/main/java/com/azure/ai/formrecognizer/perf/ReceiptRecognitionStreamTest.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.perf;
+
+import com.azure.ai.formrecognizer.models.FormRecognizerOperationResult;
+import com.azure.ai.formrecognizer.models.RecognizedForm;
+import com.azure.ai.formrecognizer.perf.core.ServiceTest;
+import com.azure.ai.formrecognizer.perf.core.Utility;
+import com.azure.core.util.polling.SyncPoller;
+import com.azure.perf.test.core.PerfStressOptions;
+import reactor.core.publisher.Mono;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
+public class ReceiptRecognitionStreamTest extends ServiceTest<PerfStressOptions> {
+    private FileInputStream fileInputStream;
+    private String receiptFilePath = "src/main/resources/samplesFiles/contoso-receipt.png";
+    private long fileLength;
+    public ReceiptRecognitionStreamTest(PerfStressOptions options) {
+        super(options);
+    }
+
+    @Override
+    public Mono<Void> globalSetupAsync() {
+        return super.globalSetupAsync().then(createReceiptInputStream());
```

Comment:
This will instantiate the FileInputStream variable for the first thread only.
Assuming you want to have a separate FileInputStream for each thread. I'd recommend instantiating this in the class constructor.

Currently, your test not work, if more than 1 thread is specified.